### PR TITLE
CI: set default timeout

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ image: "$IMAGE"
 
 default:
   interruptible: true
+  timeout: 2h
```
Comment (Zimmi48) [inline .gitlab-ci.yml:5]: The rest of this file contains timeouts expressed in minutes (like 120m). But more importantly, you didn't put a larger timeout for fiat-crypto. Is that on purpose? And shouldn't we have a smaller default timeout and more explicitly mark all the jobs that take longer? We could use some of the stats that @JasonGross created this summer to get a precise idea of what is needed for what BTW (although this was with the shared runners).

**Comment by SkySkimmer** [comment]
Updated with tighter default timeout (1h) and specific timeouts for longer jobs.

**Comment by SkySkimmer** [comment]
@coqbot run full ci

**Review by Alizter — APPROVED** [review]

**Comment by Alizter** [comment]
Are the CI failures both time outs? I cannot tell.

**Comment by SkySkimmer** [comment]
>I think it would be nice if coqbot could detect a job as timed out and report it as such in the summary.

It already does IIRC

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**